Clock.py

```diff
@@ -24,7 +24,8 @@
 SOFTWARE.
 """
 
-"""Clock.py
```

Comment:
Following the """ immediately with the file/class/method name ("""Clock.py) is done to provide Python documentation from the docstring comments. See [PEP 257](https://www.python.org/dev/peps/pep-0257/), [Documenting Python Code](https://realpython.com/documenting-python-code/#documenting-your-python-code-base-using-docstrings) and [Documenting Python](https://devguide.python.org/documenting/)

Reply:
Thank you! I didn't know that. I thought they were typos. I'll fix them before merging.

Reply:
Perfect. All good then.